src/vibe/scss/components/tabs.scss

```diff
@@ -2,140 +2,135 @@
    ----------------------------- */
 
 .tabs-container {
-	@include cardShadow;
-	border-radius: $cardRoundedCorner $cardRoundedCorner 0 0;
-	overflow: hidden;
-	margin-bottom: 20px;
+  @include cardShadow;
+  border-radius: $cardRoundedCorner $cardRoundedCorner 0 0;
+  overflow: hidden;
+  margin-bottom: 20px;
 }
 .nav-tabs {
-    background-color: $tabHeadingBgColor;
-    border-bottom: 0;
-    border-bottom: 1px solid #ebeef0;
-    overflow-x: scroll;
```

Comment:
Removing this fixes the issue. 

Everything else is prettier doing it's thing.